<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>410</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>72</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbCaminhoArquivo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>351</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Caminho arquivo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>376</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblVideoProcessado">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>420</width>
      <height>320</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>420</width>
      <height>320</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>420</width>
      <height>320</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Video Processado</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblVideoOriginal">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>420</width>
      <height>320</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>420</width>
      <height>320</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>420</width>
      <height>320</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Original</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>350</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frames Registrados:</string>
    </property>
   </widget>
   <widget class="QLabel" name="numInfracoesLabel">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>350</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>584</x>
      <y>380</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="frameCountLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>380</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1016</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
